Code/GraphMol/SynthonSpaceSearch/SynthonSpace.h

```diff
@@ -194,6 +194,20 @@ class RDKIT_SYNTHONSPACESEARCH_EXPORT SynthonSpace {
       const SubstructMatchParameters &matchParams = SubstructMatchParameters(),
       const SynthonSpaceSearchParams &params = SynthonSpaceSearchParams());
 
+  /*!
+   * Perform a substructure search with the given query molecule across
+   * the synthonspace library.  Duplicate SMILES strings produced by
+   * different reactions will be returned.
+   *
+   * @param query : query molecule
+   * @param callback: user-provided callback receiving chunks of ROMols.
```

Comment:
I think it would be very useful here if there was a brief explanation of the difference and flexibility that the callback offers.  @greglandrum may disagree, but I would be happy with a reference to an example in the test codes showing it stuffing a vector of SMILES strings.